x-pack/legacy/plugins/ml/public/management/jobs_list/components/access_denied_page.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import {
+  EuiCallOut,
+  EuiPage,
+  EuiPageBody,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+export const AccessDeniedPage = () => (
```

Comment:
This page shows up if the user is on a basic license. Just checking if the ML management menu should be hidden altogether?

Reply:
from the discussions we had, the expected behaviour was that if the licence and feature checks fail, the feature would be disabled altogether.
this is checked using `features.ml.showLinks`

Reply:
Good catch - updated in 9d707f974c167ac7d4469d64fc36a5e77653ff45